clang/test/Format/fail-on-incomplete.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: cat %s | not clang-format --fail-on-incomplete-format | FileCheck %s
+// RUN: cat %s | clang-format | FileCheck %s
+int a([) {}
+
+// CHECK: int a([) {}
```

Comment:
It's missing the EOF. Also, use a single hyphen for the option like other tests.
```suggestion
// RUN: clang-format %s
// RUN: not clang-format %s -fail-on-incomplete-format

int a(
```

Reply:
> It's missing the EOF. Also, use a single hyphen for the option like other tests.

Done. It took me quite a while but `-style=LLVM` is also required as the .clang-format in the parent directory sets DisableFormat to true. Before I didn't notice because the `cat %s | clang-format` was bypassing the lookup for the config file.

Reply:
> Done. It took me quite a while but `-style=LLVM` is also required as the .clang-format in the parent directory sets DisableFormat to true. Before I didn't notice because the `cat %s | clang-format` was bypassing the lookup for the config file.

It seems that `-style=LLVM` is needed whether the input comes from the stdin or a file.